raiko-host/src/main.rs

```diff
@@ -30,17 +30,14 @@ struct Args {
         default_value = "0.0.0.0:8080"
     )]
     /// Server bind address
-    /// [default: 0.0.0.0:8080]
```

Comment:
why remove this default values

Reply:
Sorry, that was accidental. Sorry, I will revert it back.